helpdesk/templates/teams/teamattendanceevent_form.html

```diff
@@ -0,0 +1,17 @@
+{% extends "layouts/base_app.html" %}
```

Comment:
Thought: Should this page include the full log history for a team? That way it's at least possible to see a bit more detail on a team.